components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -56,9 +56,10 @@ spec:
       - image: "{{ .Values.dashboard.image.repository }}:{{ .Values.dashboard.image.tag }}"
         imagePullPolicy: {{ .Values.dashboard.image.pullPolicy }}
         name: dashboard-{{ .Chart.Name }}
+        {{- if .Values.dashboard.containerSecurityContext }}
         securityContext:
-          runAsUser: 999
-          runAsGroup: 999
+        {{- toYaml .Values.dashboard.containerSecurityContext | nindent 12 }}
+        {{- end }}
```

Comment:
@komalsukhani I only copy pasted those out. That's why I asked the question.

Reply:
Sorry my bad. I didn't notice it. I think we should fix it.

Reply:
No worries :)